lib/elixir/lib/node.ex

```diff
@@ -257,6 +257,38 @@ defmodule Node do
     :erlang.spawn_link(node, module, fun, args)
   end
 
+  @doc """
+  Returns the PID of a new process started by the application of `fun` on `node`,
+  and a reference for a monitor created to the new process. Otherwise works
+  like `spawn/1`
+
+  If the node identified by `node` does not support distributed `spawn_monitor()`,
+  the call will fail with a `:notsup` exception.
```

Comment:
It doesn't show as `notsup` in Elixir. It will either be ErlangError or something else :)

Reply:
Do you think we should leave it undocumented or indicate thet the call will raise `%ErlangError{original::notsup}`? as There is no specification for that one in `ErlangError.normalize`. Or should we normalize it to `%NotSupportedError{}`?

Reply:
My honest opinion: we would leave it undocumented. It only happens when using distribution across Erlang nodes running in different versions and, if you are at this point, then you are likely familiar with the disclaimers that may happen.

Or perhaps, we would document like this: "This functionality was added on Erlang/OTP 23. Using this functionality to communicate with a node running on an earlier version will fail.".